ext/mbstring/tests/gh7902.phpt

```diff
@@ -3,6 +3,7 @@ GH-7902 (mb_send_mail may delimit headers with LF only)
 --SKIPIF--
 <?php
 if (!extension_loaded("mbstring")) die("skip mbstring extension not available");
+if (str_contains(getcwd(), " ")) die("skip sendmail_path ini with spaces");
```

Comment:
This test is also broken as is as a ``--EXTENSIONS--`` section with mbstring is needed. Must have been missed while merging up from 8.0 to 8.1 as this section is new since 8.1.

Reply:
I'll fix these two existing mistakes tonight